### PR TITLE
Dropped Python 3.5 from testing and packaging.

### PR DESCRIPTION
**Pull request by MatthewHambley**
As per issue #15, this change means we no longer test or package for Python 3.5.

Awkwardly this request is run under the existing regime so runs all 3. If you check the original branch, green tick above the file listing, you will see that it ran only 3.6 and 3.7.

### CODE REVIEW

File: .github/workflows/pythonpackage.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: [3.5, 3.6, 3.7]
+        python-version: [3.6, 3.7]
```
Comment (stevewardle) [inline .github/workflows/pythonpackage.yml]: Can we test 3.8 as well?

**Comment by stevewardle** [comment]
Also note - the checks should run correctly on `metomi/fab` once #19 gets committed and this is rebased

**Comment by stevewardle** [comment]
Well the fix to the testing worked, although it looks like we have a slight issue in that since this test removes testing of 3.5 the checks won't fully complete - I think once this is through review and (essentially) ready to merge we'll just de-activate the 3.5 test requirement in the settings so that we can do the commit (it'll need to be de-activated afterward anyway)

File: .github/workflows/pythonpackage.yml (hunk 1 above)
Comment (MatthewHambley) [reply .github/workflows/pythonpackage.yml]: Your wish is my command.

**Pull request merged**